CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Version 256:
 * Remove redundant template in service_base
 * Expand CI matrix using Azure Pipelines
 * Make chat websocket javascript client more user friendly
+* Don't call `allocator_traits<T>::construct` unnecessarily
```

Comment:
The change message could be "allocator_traits<T>::construct is for user-defined types"